src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -858,7 +858,10 @@ static void Main()
 " + trivial2uple + trivial3uple;
 
             CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature()).VerifyDiagnostics(
-                );
+                // (6,27): error CS8206: Tuple with '3' elements cannot be converted to type '(int, string)'.
```

Comment:
I'm not sure we need the single-quote characters around 3.
Consider `Tuple with 3 elements cannot be converted to type '(int, string)'.`